ghc-9.2.4.stack.yaml

```diff
@@ -0,0 +1,6 @@
+resolver: nightly-2022-10-11
+packages:
+  - .
```

Comment:
maybe also add 9.2 support in the ci, since you went trough the effort of figuring out the stack.yaml file.

Reply:
I bumped `nightly` to the `LTS` resolver for `ghc-9.2.5`.